kvirt/config.py

```diff
@@ -1264,7 +1275,7 @@ def revert_plan(self, plan, snapshotname=None):
 
     def plan(self, plan, ansible=False, url=None, path=None, container=False, inputfile=None, inputstring=None,
              overrides={}, info=False, update=False, embedded=False, download=False, quiet=False, doc=False,
-             onlyassets=False, pre=True):
+             onlyassets=False, pre=True, cluster=None):
```

Comment:
same comment as earlier, an additional cluster variables isnt needed

Reply:
Right ... had no idea into how `overrides` work. Still don't quite get how I would then "reach" the cluster name via the `overrides` var